components/01-atoms/controls/cta/_cta.scss

```diff
@@ -49,7 +49,7 @@ $cta-outline-weights: (
     --color-cta-bg: var(--color-cta-primary);
     --color-cta-bg-hover: transparent;
     --color-cta-border: var(--color-cta-primary);
-    --color-cta-text: var(--color-cta-secondary);
+    --color-cta-text: var(--color-gray-800);
```

Comment:
Per our zoom, this is affecting too many places. We want to only change the color schemes below.